plugins/kubernetes/app/views/samson_kubernetes/_build_show.erb

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+  <%= check_box_tag :kubernetes_job%>
+  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job", class: "control-label" %>
```

Comment:
If this is persisted as a build's parameter, it becomes a part of core Samson workflow, doesn't it?